SDWebImage/MapKit/MKAnnotationView+WebCache.m

```diff
@@ -55,14 +50,13 @@ - (void)sd_setImageWithURL:(nullable NSURL *)url
                    context:(nullable SDWebImageContext *)context
                   progress:(nullable SDImageLoaderProgressBlock)progressBlock
                  completed:(nullable SDExternalCompletionBlock)completedBlock {
-    @weakify(self);
+    __weak typeof(self) wself = self;
```

Comment:
Now the `MapKit` component can only dependent the public header from `SDWebImage`, not the private header. So this can not use `@weakify` macro